Octokit/Models/Request/SearchCodeRequest.cs

```diff
@@ -159,9 +159,9 @@ public override IReadOnlyList<string> MergedQualifiers()
                     string.Join(",", In.Select(i => i.ToParameter()))));
             }
 
-            if (Language != null)
+            if (Language.HasValue && !string.IsNullOrWhiteSpace(Language.Value.StringValue))
```

Comment:
I don't think these needs to be nullable. The null or whitespace check on `StringValue` should be enough.